subprojects/gradle.properties

```diff
@@ -18,3 +18,7 @@ android.useAndroidX=true
 projectVersion=2021.3
 # Disable console output https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/202
 systemProp.dependency.analysis.silent=true
+# Versions
+systemProp.kotlinVersion=1.4.21
+systemProp.detektVersion=1.15.0
+systemProp.androidGradlePluginVersion=4.1.1
```

Comment:
Can't we use project properties instead of system properties? Are there any restrictions?

Reply:
don't know, i just reverted it back